Beecrowd/1069.c

```diff
@@ -0,0 +1,43 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+int main(){
+    int num_linha, tamanho_mina, inicio_diamante, fim_diamante, diamantes, contador;
+    char mina[1001];
+
+    scanf("%d", &num_linha);
+
+    while(num_linha--){
+        diamantes = 0;
+        contador = 0;
+        inicio_diamante = -1;
+        fim_diamante = -1;
+
+        scanf("%s", mina);
+
+        tamanho_mina = strlen(mina);
+        mina[tamanho_mina] = '\0';
```

Comment:
Curiosidade: A maneira como a função `strlen` funciona, é exatamente contando o numero de caracteres até encontrar o `\0` na string, ou seja essa segunda linha é desnecessária, pq `mina[tamanho_mina]` já é igual a `\0` por definição:
https://github.com/lattera/glibc/blob/master/string/strlen.c#L30-L108

```suggestion
        tamanho_mina = strlen(mina);
```